src/main/java/org/opentripplanner/updater/bike_rental/GenericJsonBikeRentalDataSource.java

```diff
@@ -55,6 +61,9 @@ public GenericJsonBikeRentalDataSource(RentalUpdaterError.Severity severityFailu
 
     /**
      *
+     * @param severityFailureType The severity level to create an error with when fetching from this particular data
```

Comment:
Suggestion: "The security level *given to an error* when...".

Reply:
Refactored in 19d09a6